packages/react/src/Image/Image.features.stories.tsx

```diff
@@ -66,7 +78,7 @@ export const CustomClassWithAspectRatio: StoryFn<typeof Image> = () => (
   <Image
     src={placeholderImage}
     className={style['custom-image']}
-    alt="placeholder, blank area with an off-white background color"
+    alt="placeholder, blank area with an off-white background color, custom aspect ratio"
```

Comment:
Based on the feedback you received, we've still go an issue with these stories with custom red outlines:

> (sev 3) Within the [Custom Class with Aspect Ratio](https://primer.style/brand/storybook/?path=/story/components-image-features--custom-class-with-aspect-ratio) and the [Custom Class on Picture](https://primer.style/brand/storybook/?path=/story/components-image-features--custom-class-on-picture) examples, the alt attribute has the same placeholder value of "placeholder, blank area with an off-white background color" **although a red frame outlines the image.**

>  **although a red frame outlines the image.**

Can we update the alt text to indicate there's a red outline here please?

Also FYI that the height and width attributes here are still `100%`
![Screenshot 2023-07-31 at 15 03 06](https://github.com/primer/brand/assets/13340707/3ad584be-27dc-46d0-8cb4-ae9308c107e8).

This likely needs a code change under-the-hood for this component. How much effort is that?

Reply:
I discussed the 100% height and width with the a11y eng team and that is ok to keep in there for functionality. I will add the red outline to the description.